spec/helpers/createComponent.js

```diff
@@ -1,11 +1,7 @@
-var React = require('react/addons');
-var TestUtils = React.addons.TestUtils;
-
-// Thanks for sharing, Simon Smith.
-// https://gist.github.com/simonsmith/b478d6166acd57829d15
```

Comment:
thanks again @simonsmith for leaving the comment on the snippet about `skin-deep` that saved quite some time :wink: